fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/FHIRDbFlavorImpl.java

```diff
@@ -6,6 +6,7 @@
 
 package com.ibm.fhir.persistence.jdbc.connection;
 
+import com.ibm.fhir.database.utils.api.SchemaType;
```

Comment:
update copyright.  I didn't check it for most files, but noticed it on this one...probably worth a big grep to find others.  something like this:
```
for file in $(git diff origin/main..HEAD --name-status | cut -f2); do grep -n 'Copyright' $file /dev/null | grep '\(C\) Copyright IBM' | grep -v 2022; done
```